script/translations/develop.py

```diff
@@ -105,10 +104,7 @@ def run_single(translations, flattened_translations, integration):
         integration_strings, flattened_translations
     )
 
-    if download.DOWNLOAD_DIR.is_dir():
-        rmtree(str(download.DOWNLOAD_DIR))
-
-    download.DOWNLOAD_DIR.mkdir(parents=True)
+    download.DOWNLOAD_DIR.mkdir(parents=True, exist_ok=True)
```

Comment:
This is not a fix for the dev container (as the PR implies), and should thus not be part of this PR.

Reply:
should I make a separate PR for this? the issue prevents starting the debug session because of the failing task:
```
Generating translations for tilt_ble
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "/workspaces/core/script/translations/__main__.py", line 30, in <module>
    sys.exit(main())
             ^^^^^^
  File "/workspaces/core/script/translations/__main__.py", line 25, in main
    return module.run()
           ^^^^^^^^^^^^
  File "/workspaces/core/script/translations/develop.py", line 127, in run
    run_single(translations, flattened_translations, integration)
  File "/workspaces/core/script/translations/develop.py", line 108, in run_single
    rmtree(str(download.DOWNLOAD_DIR))
  File "/usr/local/lib/python3.12/shutil.py", line 759, in rmtree
    _rmtree_safe_fd(stack, onexc)
  File "/usr/local/lib/python3.12/shutil.py", line 703, in _rmtree_safe_fd
    onexc(func, path, err)
  File "/usr/local/lib/python3.12/shutil.py", line 680, in _rmtree_safe_fd
    raise OSError("Cannot call rmtree on a symbolic link")
OSError: [Errno None] None: '/workspaces/core/build/translations-download'
```

https://github.com/user-attachments/assets/c64e262c-a2f3-4846-8e4c-ac35621b7533

Reply:
It is a code change unrelated to the dev container, thus should be in e different / its own PR.

Reply:
moved to https://github.com/home-assistant/core/pull/127675